Source/GSDisplayServer.m

```diff
@@ -601,6 +601,12 @@ - (void) miniwindow: (int) win
   [self subclassResponsibility: _cmd];
 }
 
+/** Hides all application windows */
```

Comment:
The comment should be something like this:
Ask the window manager to hide all the application windows for us. Return whether they have been hidden.

Reply:
Fixed.